source/tutorial/backend.txt

```diff
@@ -127,25 +135,43 @@ To create the endpoint:
 
 #. Click :guilabel:`Save` and deploy the endpoint.
 
-.. tip::
+#. Under :guilabel:`Operation Type`, copy the endpoint callback URL. You'll need it 
+   to set up the GitHub webhook later.
+
+.. _function-authentication-settings:
+
+Update Your Endpoint Function's Authentication Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Now that you've created the endpoint in your app, you need to change the authentication
+settings for the endpoint's function so that the GitHub webhook is accepted.
+
+To update your function's authentication settings:
 
-   Copy the endpoint URL from the configuration page. You'll need it in
-   the next step.
+1. In the {+leftnav+}, click :guilabel:`Functions`.
+
+#. Find your endpoint function and select it.
+
+#. Click :guilabel:`Settings`.
+
+#. Under :guilabel:`Authentication`, change the authentication method to :guilabel:`System`.
+
+#. Click :guilabel:`Save` and deploy the function.
 
 Connect the Endpoint to a GitHub Webhook
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Now that you've created the endpoint in your app, you need to set up a
+With , you need to set up a
```

Comment:
missing something here

Reply:
Huh. I wonder how I left that out. I've included for my latest commit. Thanks for noticing!